templates/web/pages/product_edit/product_edit_form_display.tt.js

```diff
@@ -34,3 +34,21 @@
 		\$('#nutrition_data_div').show();
 	}
 });
+
+  \$('#nutriment_sugars').on('input', function() {
+    var inputValue = $(this).val();
```

Comment:
hey @stephanegigandet @yuktea 
I jQuery is blocking this from working 

its like the `.val()` is not a function.   `Uncaught TypeError: this.val is not a function`

think i have called the element correctly so it must be working

Reply:
Hi @jnsereko , it's because this file is a template file, so anything starting with $ is interpreted as a variable in the template. If you want the $ to be output in the JS code, you need to escape it like this: \$

Reply:
thanks @stephanegigandet